File: PreferencesButton.m
//
//  PreferencesButton.m
//  Password Factory
//
//  Created by Cristiana Yambo on 8/16/15.
//  Copyright (c) 2015 c13. All rights reserved.
//

#import "PreferencesButton.h"
#import "StyleKit.h"
@implementation PreferencesButton
-(instancetype)initWithCoder:(NSCoder *)coder {
    self = [super initWithCoder:coder];
    self.bordered = NO;
    self.imagePosition = NSImageOnly;
    [self setButtonType:NSMomentaryChangeButton];
    self.bezelStyle = NSRegularSquareBezelStyle;
    self.image = [StyleKit imageOfPreferencesButton];
    return self;
    
}

- (void)drawRect:(NSRect)dirtyRect {
    [super drawRect:dirtyRect];
    
    // Drawing code here.
}

@end

File: Password Factory iOS/Controls/Buttons/PreferencesButton.swift
//
//  PreferencesButton.swift
//  Password Factory iOS
//
//  Created by Cristiana Yambo on 12/16/17.
//  Copyright © 2017 Cristiana Yambo. All rights reserved.
//


import UIKit
@IBDesignable

/// Preferences Button
class PreferencesButton: UIBarButtonItem {
    override func prepareForInterfaceBuilder() {
        super.prepareForInterfaceBuilder()
        setDisplay()
    }
    override func awakeFromNib() {
        super.awakeFromNib()
        setDisplay()
    }
    func setDisplay() {
        image = StyleKit.imageOfPreferencesButton(strokeColor: UIColor.black)
        title = ""
    }
    
}
